libraries/botbuilder/src/eventFactory.ts

```diff
@@ -33,7 +32,7 @@ export class EventFactory {
         const handoffEvent = this.createHandoffEvent(
             HandoffEventNames.InitiateHandoff,
             handoffContext,
-            context.activity.conversation as ConversationAccount // TODO(joshgummersall) why is this necessary?
+            context.activity.conversation
```

Comment:
Does this compile?! Amazing.

Reply:
Yeah! I saw the comment and was confused as well. Don't know why we needed it in the first place....